Fabric Bridge mod causes server crash

### Stacktrace

```
[29.01 20:49:21.024] INFO: [fabric-ffa-1] Caused by: net.fabricmc.loader.impl.metadata.ParseMetadataException: Error reading fabric.mod.json file for mod at /home/cloudnet/temp/services/fabric-ffa-1_f629a864-81f0-4898-9935-521f3b6755e2/mods/cloudnet-bridge.jar: Invalid mod id CloudNet_Bridge:
[29.01 20:49:21.024] INFO: [fabric-ffa-1]       - It starts with an invalid character 'C' (it must be a lowercase a-z - uppercase isn't allowed anywhere in the ID)
[29.01 20:49:21.025] INFO: [fabric-ffa-1]       - It contains invalid characters: 'B', 'N'!
[29.01 20:49:21.025] INFO: [fabric-ffa-1]       at net.fabricmc.loader.impl.metadata.MetadataVerifier.checkModId(MetadataVerifier.java:146) ~[fabric-loader-0.14.13.jar:?]
[29.01 20:49:21.026] INFO: [fabric-ffa-1]       at net.fabricmc.loader.impl.metadata.MetadataVerifier.verify(MetadataVerifier.java:52) ~[fabric-loader-0.14.13.jar:?]
[29.01 20:49:21.026] INFO: [fabric-ffa-1]       at net.fabricmc.loader.impl.metadata.ModMetadataParser.parseMetadata(ModMetadataParser.java:49) ~[fabric-loader-0.14.13.jar:?]
[29.01 20:49:21.026] INFO: [fabric-ffa-1]       at net.fabricmc.loader.impl.discovery.ModDiscoverer$ModScanTask.parseMetadata(ModDiscoverer.java:469) ~[fabric-loader-0.14.13.jar:?]
[29.01 20:49:21.027] INFO: [fabric-ffa-1]       at net.fabricmc.loader.impl.discovery.ModDiscoverer$ModScanTask.computeJarFile(ModDiscoverer.java:291) ~[fabric-loader-0.14.13.jar:?]
[29.01 20:49:21.027] INFO: [fabric-ffa-1]       at net.fabricmc.loader.impl.discovery.ModDiscoverer$ModScanTask.compute(ModDiscoverer.java:253) ~[fabric-loader-0.14.13.jar:?]
[29.01 20:49:21.028] INFO: [fabric-ffa-1]       at net.fabricmc.loader.impl.discovery.ModDiscoverer$ModScanTask.compute(ModDiscoverer.java:212) ~[fabric-loader-0.14.13.jar:?]
[29.01 20:49:21.058] INFO: [fabric-ffa-1]       at java.util.concurrent.RecursiveTask.exec(RecursiveTask.java:100) ~[?:?]
[29.01 20:49:21.069] INFO: [fabric-ffa-1]       at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
[29.01 20:49:21.075] INFO: [fabric-ffa-1]       at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
[29.01 20:49:21.076] INFO: [fabric-ffa-1]       at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
[29.01 20:49:21.076] INFO: [fabric-ffa-1]       at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
[29.01 20:49:21.077] INFO: [fabric-ffa-1]       at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
```

### Actions to reproduce

Start a fabric 1.19.3 server

### CloudNet version

```
[29.01 20:51:40.760] INFO:
[29.01 20:51:40.760] INFO: CloudNet Blizzard 4.0.0-RC7-SNAPSHOT e3b4bffb
[29.01 20:51:40.761] INFO: Discord: <https://discord.cloudnetservice.eu/>
[29.01 20:51:40.761] INFO:
[29.01 20:51:40.761] INFO: ClusterId: bbb5ac76-****-4871-****-ca1949748f21
[29.01 20:51:40.761] INFO: NodeId: Main-Node
[29.01 20:51:40.761] INFO: Head-NodeId: Main-Node
[29.01 20:51:40.761] INFO: CPU usage: (P/S) 3.85/100/100%
[29.01 20:51:40.761] INFO: Node services memory allocation (U/R/M): 22828/22828/126797 MB
[29.01 20:51:40.761] INFO: Threads: 100
[29.01 20:51:40.762] INFO: Heap usage: 76/256MB
[29.01 20:51:40.762] INFO: JVM: Oracle Corporation 17 (Java HotSpot(TM) 64-Bit Server VM 17.0.1+12-LTS-39)
[29.01 20:51:40.762] INFO: Update Repo: CloudNetService/launchermeta, Update Branch: nightly
[29.01 20:51:40.779] INFO:
```

### Other

Is there an option to disable the bridge mod getting copied into the server? cuz the server crashes continiously 
![image](https://user-images.githubusercontent.com/63669478/215352248-7e875937-f615-46c0-8e40-b4de738e9684.png)


### Issue uniqueness

- [X] Yes, this issue is unique. There are no similar issues.

--- a/ext/platform-inject-support/processor/src/main/java/eu/cloudnetservice/ext/platforminject/processor/platform/fabric/FabricPluginInfoGenerator.java
+++ b/ext/platform-inject-support/processor/src/main/java/eu/cloudnetservice/ext/platforminject/processor/platform/fabric/FabricPluginInfoGenerator.java
@@ -31,14 +31,19 @@ import lombok.NonNull;
 
 final class FabricPluginInfoGenerator extends NightConfigInfoGenerator {
 
-  // https://fabricmc.net/wiki/documentation:fabric_mod_json#mandatory_fields
+  // [a-z][a-z0-9-_]{1,63}
   private static final PluginIdGenerator MOD_ID_GENERATOR = PluginIdGenerator.withBoundedLength(1, 63)
     .registerRange(
       0,
+      1,
+      'a',
+      CharRange.range('a', 'z'))
+    .registerRange(
+      1,
       '_',
       CharRange.range('_'),
+      CharRange.range('-'),
       CharRange.range('a', 'z'),
-      CharRange.range('A', 'Z'),
       CharRange.range('0', '9'));
 
   public FabricPluginInfoGenerator() {

